libs/cordova.force.js

```diff
@@ -532,6 +532,8 @@ cordova.define("com.salesforce.plugin.smartstore", function (require, exports, m
 
     var exec = require("com.salesforce.util.exec").exec;
 
+    var defaultStoreName = "defaultStore";
```

Comment:
This has to match what's defined in `native`, correct? Can we remove this, so `native` is the single source of truth and we don't have to maintain them in both?